packages/workbox-build/src/lib/errors.js

```diff
@@ -32,7 +32,7 @@ module.exports = {
   'unable-to-copy-workbox-sw': ol`workbox-sw is needed by the service worker
```

Comment:
Some of these messages need to be pruned, as they're handled implicitly by `joi` validation.